web/v1/snowflake/sql-runner/sql/standard/99-migrations/1.0.3-migration/alter-tables.sql

```diff
@@ -0,0 +1,364 @@
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN NETWORK_USERID VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_TITLE VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_URL VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_URLSCHEME VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_URLHOST VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_URLPATH VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_URLQUERY VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN FIRST_PAGE_URLFRAGMENT VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_TITLE VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_URL VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_URLSCHEME VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_URLHOST VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_URLPATH VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_URLQUERY VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN LAST_PAGE_URLFRAGMENT VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFERRER VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_URLSCHEME VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_URLHOST VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_URLPATH VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_URLQUERY VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_URLFRAGMENT VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_MEDIUM VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_SOURCE VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN REFR_TERM VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_MEDIUM VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_SOURCE VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_TERM VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_CONTENT VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_CAMPAIGN VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_CLICKID VARCHAR(16777216);
+ALTER TABLE {{.output_schema}}.USERS_WEBTEST MODIFY COLUMN MKT_NETWORK VARCHAR(16777216);
```

Comment:
@agnessnowplow why is this `VARCHAR(16777216)` instead of just `VARCHAR`?

Reply:
It is effectively the same as that is the limit for Snowflake, I can modify it, if it helps avoiding confusion?

Reply:
I think it would help - if I have that question customers will as well!